### PR TITLE
Feature: fixed item width

### PR DESCRIPTION
**Pull request by nicolas-cusan**
Hello @jamiebuilds, first of all great library!

I am using it in a project that requires elements to have a fixed width. I have added this option via a prop named `fixedItemSize` wich takes a number (pixels) and simply replaces `1r` with `fixedItemSize` in the `grid-template-columns` of the grid container.

To run the example use `yarn start start-fixed-width` (the npm script names and the folder name could be nicer of course).

Thank you for having a look!

### CODE REVIEW

**Comment by jamiebuilds** [comment]
Could you put the fixed width example on the same page as the existing example. I'd prefer not to have entirely separate apps for the examples

**Comment by nicolas-cusan** [comment]
Hi @jamiebuilds I integrated a switch on the example page that lets you toggle between the two modes (fixed and variable).

Thank you!

**Comment by nicolas-cusan** [comment]
@jamiebuilds any news on this? Thank you!

**Comment by jamiebuilds** [comment]
This doesn't quite work well. `getColumnCount` in the example doesn't account for grid gap size. That wasn't a problem before because 1fr will shrink items too, but fixed size won't

File: src/GridList.tsx [hunk 1]
```diff
@@ -496,7 +499,7 @@ export default function GridList<P>(props: GridListProps<P>) {
 						display: "grid",
 						gridTemplateColumns:
 							configData !== null
-								? `repeat(${configData.columnCount}, 1fr)`
+								? `repeat(${configData.columnCount}, ${colWidth})`
 								: undefined,
 						gridGap: configData ? configData.gridGap : undefined,
 						alignItems: "center",
```
Comment (jamiebuilds) [inline src/GridList.tsx:509]: Mind also adding `justifyContent: "center"` here

**Pull request merged**